python/python/benchmarks/test_search.py

```diff
@@ -75,6 +93,7 @@ def create_base_dataset(data_dir: Path) -> lance.LanceDataset:
 
     dataset.create_scalar_index("filterable", "BTREE")
     dataset.create_scalar_index("category", "BITMAP")
+    dataset.create_scalar_index("genres", "TAG")
```

Comment:
In Postgres, these are called "GIN" or "Generalized Inverted Index". Could call it "INVERTED_STRING_INDEX" or something like that. IMO it's nice to re-use existing names for concepts rather than create new names.

Reply:
Why is the FTS index called inverted index? I wonder if we should move these names around.

Reply:
I'm fine with label list index.

Reply:
> Why is the FTS index called inverted index? I wonder if we should move these names around.

"inverted index" is a very general concept.  Normally you have a structure that goes from row id to value.  "Inverted index" just menas you have a structure that goes from value to row id.  Pretty much every index is an inverted index.  Inverted indices are often used for full text search.  E.g. TF-IDF and BM25 are both "inverted indices".

IVF is an inverted index.

Also, GIN, in postgres, is much more general (e.g. it can be used for full text search on a string column) and I'm not sure a GIN index on an Array column is doing the same thing as we are doing here.

Reply:
Ok, yeah, reading more, I suppose one could think of this as a GIN index.